### PR TITLE
Added new build live stream event

### PR DESCRIPTION
**Pull request by KrisvanderMast**

### CODE REVIEW

File: _posts/events/2018-02-05---Enter-The-Matrix-Securing-Microsoft-Azure-Assets.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ---
 layout: single
 title: "2018-02-05 - Enter The Matrix: Securing Microsoft Azure’s assets"
+<<<<<<< HEAD
```
Comment (tomkerkhove) [inline _posts/events/2018-02-05---Enter-The-Matrix-Securing-Microsoft-Azure-Assets.md]: Merge conflicts

File: _posts/events/2018-05-07---build-2018-live-stream.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
 ---
 layout: single
 title: "2018-05-07 - BUILD 2018 live stream"
+<<<<<<< HEAD
```
Comment (tomkerkhove) [inline _posts/events/2018-05-07---build-2018-live-stream.md]: Merge conflicts

File: _posts/events/2018-05-07---build-2018-live-stream.md [hunk 3]
```diff
@@ -13,12 +17,27 @@ redirect_from:
  - /events/2018/05/07/build-2018-live-stream.html
 ---
 
+<<<<<<< HEAD
```
Comment (tomkerkhove) [inline _posts/events/2018-05-07---build-2018-live-stream.md]: Merge conflicts

File: _posts/events/2018-05-07---build-2018-live-stream.md [hunk 4]
```diff
@@ -13,12 +17,27 @@ redirect_from:
...
+The developers conference we're all been waiting for is happening again in May. We're talking about Microsoft //BUILD of course. As not everyone can attend in person we thought it would be a great idea to partner with our friends of MADN, OWIC and VISUG and have a live stream. Microsoft Belgium opens up the doors for us. 
+=======
 The developers conference we've all been waiting for is happening again in May. We're talking about Microsoft //BUILD of course. As not everyone can attend the real version in the USA in person, we thought it would be a great idea to partner with our friends of [MADN](http://www.madn.be), [OWIC](http://www.owic.be) and [VISUG](http://www.visug.be) to organize a local live stream event. Microsoft Belgium opens up their doors for us. 
+>>>>>>> upstream/master
 
 ## Agenda
 //BUILD live stream, food and networking with friends and peers.
 
 ## Practical details
+<<<<<<< HEAD
```
Comment (tomkerkhove) [inline _posts/events/2018-05-07---build-2018-live-stream.md]: Merge conflicts

**Comment by maartenba** [comment]
Will cherry-pick the changes, but not merging as-is because of conflicts (and excessive history).

**Pull request closed without merging**